Sources/PipecatClientIOSDaily/util/DailyTransportConnectionParams.swift

```diff
@@ -1,20 +1,21 @@
 import Foundation
+import PipecatClientIOS
 
-struct DailyTransportAuthBundle: Decodable {
+public struct DailyTransportConnectionParams: TransportConnectionParams {
```

Comment:
one of the other major changes i made to the daily transport is that the connection params extends the actual daily `DailyCallOptions`. does that make sense here to? (or maybe you are doing that but i just don't understand swift enough)